noir_by_example/toml_versions.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+file_prefix="$1"
```

Comment:
Looks like this is not used anywhere now. Maybe it was intended to be used with -iname ?